client/tests/integration/permissions.rs

```diff
@@ -313,16 +313,17 @@ fn stored_vs_granted_token_payload() -> Result<()> {
         .expect("Failed to register mouse");
 
     // Allow alice to mint mouse asset and mint initial value
+    // Introducing some whitespaces
+    // This way, if the executor compares just JSON strings, this test would fail
+    let value_json: Value = serde_json::de::from_str(
+        &*r#"{ "asset"    :   "xor#wonderland#{id}" }"#
+            .replace("{id}", mouse_id.to_string().as_str()),
+    )
+    .unwrap();
+
```

Comment:
this is still not good because in the parsed form you've removed the whitespaces

Reply:
> create `JsonString` directly from `String` to have extra whitespaces

My main idea is to forbid creatin JsonString from a raw string to avoid invalid Jsons. Only new(), from_str(), parse::<JsonString>() are allowed because they implement ser/de chain "str->Value->String"

Reply:
Returned all back